src/core/Object3D.js

```diff
@@ -40,20 +40,24 @@ THREE.Object3D = function () {
 	Object.defineProperties( this, {
 		position: {
 			enumerable: true,
+			configurable: true,
```

Comment:
Is this just so we can do `delete this.position` later? Or is it needed for redefining it too?